sites/svelte.dev/src/routes/docs.html/+server.js

```diff
@@ -0,0 +1,2 @@
+export const prerender = true;
+export { GET } from '../docs/+server.js';
```

Comment:
I don't think we should make `docs.html` return anything. It returns "500 | not found" right now, which  is really weird, but we should just fix that to return 404

Reply:
I was trying to get it to output the same prerendered files as it did before, not sure why it's a 500/404 now. Some manifest thing maybe?

Reply:
probably https://github.com/sveltejs/kit/pull/10565 will fix the 500/404 thing

https://svelte.dev/docs.html doesn't currently return anything, but would with this PR, so this isn't really making it equivalent to what it was before